src/content/en/updates/2017/12/chrome-loadtimes-deprecated.md

```diff
@@ -0,0 +1,358 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The non-standard chrome.loadTimes() API will be deprecated in Chrome 64 now that standards-based equivalents exist for all of its features.
+
+{# wf_updated_on: 2017-12-14 #}
+{# wf_published_on: 2017-12-14 #}
+{# wf_tags: performance #}
+{# wf_blink_components: Blink>PerformanceAPIs #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: The non-standard chrome.loadTimes() API will be deprecated in Chrome 64 now that standards-based equivalents exist for all of its features. #}
+{# wf_blink_components: Blink>PerformanceAPIs #}
+
+# Chrome 64 to deprecate the chrome.loadTimes() API {: .page-title }
+
+{% include "web/_shared/contributors/philipwalton.html" %}
+
+`chrome.loadTimes()` is a non-standard API that exposes loading metrics and
+network information to developers in order to help them better understand their
+site's performance in the real world.
+
+Since this API was implemented in 2009, all of the information it reports can be
+found in standardized APIs such as [Navigation Timing 2](https://www.w3.org/TR
+/navigation-timing-2/), [Paint Timing](https://www.w3.org/TR/paint-timing/), and
+the recent [`nextHopProtocol`](https://www.w3.org/TR/resource-timing-2/#dom-
+performanceresourcetiming-nexthopprotocol) addition to Navigation Timing 2 and
+[Resource Timing 2](https://www.w3.org/TR/resource-timing-2/).
+
+These standardized APIs are being implemented by multiple browser vendors. As a
+result, `chrome.loadTimes()` is being deprecated in Chrome 64.
+
+## The deprecated API
+
+The `chrome.loadTimes()` function returns a single object containing all of its
+loading and network information. For example, the following object is the result
+of calling `chrome.loadTimes()` on [www.google.com](https://www.google.com):
```

Comment:
If you want a blazingly fast site that's more relevant to current Dev Rel goals, I suggest Paul's [Media PWA demo](https://biograf-155113.appspot.com/).

Reply:
This was mainly to show the format of the object. The site or its speed don't matter.